### ISSUE TITLE
RCNN stage cannot converge on Scannet dataset

### ISSUE THREAD

**Issue by BigDeviltjj** [sshaoshuai/PointRCNN#64]
HI, we are training Scannet dataset using pointRCNN, training data is generated by extracting point clouds from rgbd images in Scannet. Currently, we only train chair class.

The problem is RPN stage can converge correctly, but when training RCNN stage, the loss is stable at around 3 and does not decrease, but when we use the same code to train KITTI dataset, loss decreases quickly. Do you have any idea about it?

Btw, since chairs have great intra-class variance and occlusion(by tables or other things), even though RPN stage can converge, evaluation score is still much lower than kitti result. Could the problem stems from poor RPN part performance?

**Comment by sshaoshuai** [comment]
I think a little poor RPN model will not cause this problem, and you could check the ratio of postive and negative samples of the RCNN stage. 
The orientation of chair is not well-defined, maybe you could pay more attention to the canonical transformation (rotation ) or other parts related to the orientation, and check the regression target to see whether it is normal.

**Comment by BigDeviltjj** [comment]
@sshaoshuai thank you so much, I conducted more experiments afterwards, finding that when training more epochs, the loss decreases to around 2 and inference results are decent even though training loss is higher than that of training KITTI.

And the orientation problem, I solved it by associate chairs' bounding box with corresponding CAD model in ShapeNet. I'll close the issue.
